runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2872,10 +2872,10 @@ gcInitializeDefaults(J9JavaVM* vm)
 	extensions->setOmrVM(vm->omrVM);
 	vm->omrVM->_gcOmrVMExtensions = (void *)extensions;
 	vm->gcExtensions = vm->omrVM->_gcOmrVMExtensions;
-#if defined(J9VM_ENV_DATA64)
+#if defined(J9VM_GC_ENABLE_SPARSE_HEAP_ALLOCATION)
 	vm->isIndexableDualHeaderShapeEnabled = FALSE;
```

Comment:
This should be true. It was toggle on a while back as part of Dual Header shape [PR](https://github.com/eclipse-openj9/openj9/commit/15d2d84a946058e40fa221e78ff52c1c0ba0a4fc).